src/Room/Room.js

```diff
@@ -0,0 +1,24 @@
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, Text, View, Image } from "react-native";
+
+export default Room = ({ navigation }) => {
+    return (
+        <View style={styles.container}>
+            <Text>Living Room</Text>
+            <Image source={require("/assets/roomImages/livingroom.png")} style={{width: 100% , height: 100%}}></Image>
```

Comment:
This is great, I'm assuming we will be dynamically passing this image based on which Room is fetched, right?